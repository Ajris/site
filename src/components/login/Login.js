import React from 'react';
import "bootstrap/dist/css/bootstrap.min.css";
import './Login.css';
import Background from './bg-01.jpg';

export default class Login extends React.Component {
    render() {
        return (
            <div style={{backgroundImage:`url(${Background})`}}>
                <div className="container-login100">
                    <div className="wrap-login100 ">
				        <span className="login100-form-title pb-3">
					        Account Login
				        </span>
                        <form className="login100-form">
                            <div className="wrap-input100">
                                <input className="input100" type="text" name="username" placeholder="User name"/>
                            </div>
                            <div className="wrap-input100">
                                <input className="input100" type="password" name="pass" placeholder="Password"/>
                            </div>
                            <div className="container-login100-form-btn py-2">
                                <button className="login100-form-btn">
                                    Login
                                </button>
                            </div>
                        </form>
                    </div>
                </div>
            </div>

        )
    }
}