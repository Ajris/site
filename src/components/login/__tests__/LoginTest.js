import React from "react";
import { create } from "react-test-renderer";
import Login from "../Login";

describe("Login component", () => {
    test("it matches the snapshot", () => {
        const component = create(<Login/>);
        expect(component.toJSON()).toMatchSnapshot();
    });
});