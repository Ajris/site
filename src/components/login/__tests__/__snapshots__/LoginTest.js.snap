// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component it matches the snapshot 1`] = `
<div
  style={
    Object {
      "backgroundImage": "url(bg-01.jpg)",
    }
  }
>
  <div
    className="container-login100"
  >
    <div
      className="wrap-login100 "
    >
      <span
        className="login100-form-title pb-3"
      >
        Account Login
      </span>
      <form
        className="login100-form"
      >
        <div
          className="wrap-input100"
        >
          <input
            className="input100"
            name="username"
            placeholder="User name"
            type="text"
          />
        </div>
        <div
          className="wrap-input100"
        >
          <input
            className="input100"
            name="pass"
            placeholder="Password"
            type="password"
          />
        </div>
        <div
          className="container-login100-form-btn py-2"
        >
          <button
            className="login100-form-btn"
          >
            Login
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
